src/common/connection.ts

```diff
@@ -36,29 +36,67 @@ export interface ConnectionOptions {
  */
 export type ConnectionUserOptions = Partial<ConnectionOptions>
 
+/**
+ * Ledger is used to store and reference ledger information that has been
+ * captured by the Connection class.
+ */
+class Ledger {
+  private availableVersions = new RangeSet()
+  latestVersion: null | number = null
+  feeBase: null | number = null
+  feeRef: null | number = null
+
+  hasVersions(lowVersion: number, highVersion: number): boolean {
```

Comment:
Please doc, it's unclear what a version is. 

Are low / high inclusive or exclusive?

Reply:
should be inclusive, I believe